pdns/dnsdistdist/docs/changelog.rst

```diff
@@ -1,6 +1,31 @@
 Changelog
 =========
 
+.. changelog::
+  :version: 1.8.4
+  :released: 20th of September 2024
+
+  Please review the :doc:`Upgrade Guide <../upgrade_guide>` before upgrading from versions < 1.8.x.
+
+  .. change::
+    :tags: Bug Fixes
+    :pullreq: 14467
+
+    Fix a compilation issue with clang by switching to pdns::views::UnsignedCharView
+
+  .. change::
+    :tags: Bug Fixes
+    :pullreq: 14680
+    :tickets: 14562
+
+    Fix build with boost 1.86.0
+
+  .. change::
+    :tags: Bug Fixes, DNS over TLS
+    :pullreq: 14679
+
+    Prevent a data race in incoming DNS over TLS connections by storing the `OpenSSLTLSIOCtx` in the connection
```

Comment:
```suggestion
    Prevent a data race in incoming DNS over TLS connections by storing the ``OpenSSLTLSIOCtx`` in the connection
```

Reply:
Good catch, thanks! I did check the rendering in the PDF but apparently a single backtick somehow works there :man_shrugging:

Reply:
I see it in italics in both PDF and HTML with the single ticks, which is not terrible, but not what I'd choose

Reply:
For the record, I was wrong! Single tick renders in italics, double tick as a "code" style. Note to self: double-check that the PDF was actually re-generated next time.